dev/io.openliberty.jaxws.config_fat/test-applications/testWebServiceClient/src/io/openliberty/samples/jaxws/client/ClientSayHelloWithHandlerService.java

```diff
@@ -19,20 +19,30 @@
 import javax.xml.ws.WebEndpoint;
 import javax.xml.ws.WebServiceClient;
 
+import io.openliberty.jaxws.fat.stubclient.client.SimpleEchoService;
+
 @WebServiceClient(name = "SayHelloService",
-                  targetNamespace = "http://jaxws.samples.openliberty.io/",
-                  wsdlLocation = "META-INF/resources/wsdl/SayHelloService.wsdl")
+                  targetNamespace = "http://jaxws.samples.openliberty.io/")
 public class ClientSayHelloWithHandlerService extends Service {
 
     private final static URL SAYHELLOSERVICE_WSDL_LOCATION;
-
+    private static Logger logger = Logger.getLogger(ClientSayHelloWithHandlerService.class.getName());
+    
     static {
         URL url = null;
+        String urlString = null;
         try {
             URL baseUrl = ClientSayHelloWithHandlerService.class.getClassLoader().getResource(".");
-            url = new URL(baseUrl, "SayHelloService.wsdl");
+            String host = System.getProperty("hostName");
+            if (host == null) {
+                logger.info("Failed to obtain host from system property, hostName, falling back to localhost");
+                host = "localhost";
+            }
+            urlString = new StringBuilder().append("http://" + host + ":").append(Integer.getInteger("bvt.prop.HTTP_default")).append("/testWebServiceClient/SayHelloService?wsdl").toString();   
+            logger.info("URL : " + urlString + " will be used for ClientSayHelloWithHandlerService.");
+            url = new URL(baseUrl, urlString);
```

Comment:
You can should just call `URL(urlString)`

Reply:
It worked both on my local and PB. But I can change it.